http_client.rst

```diff
@@ -5,95 +5,115 @@
 The HttpClient Component
 ========================
 
-    The HttpClient component is a low-level HTTP client with support for both
-    PHP stream wrappers and cURL. It provides utilities to consume APIs and
-    supports synchronous and asynchronous operations.
-
 .. versionadded:: 4.3
 
     The HttpClient component was introduced in Symfony 4.3.
 
 Installation
 ------------
 
+The HttpClient component is a low-level HTTP client with support for both
+PHP stream wrappers and cURL. It provides utilities to consume APIs and
+supports synchronous and asynchronous operations. You can install it with:
+
 .. code-block:: terminal
 
     $ composer require symfony/http-client
 
-.. include:: /components/require_autoload.rst.inc
-
 Basic Usage
 -----------
 
 Use the :class:`Symfony\\Component\\HttpClient\\HttpClient` class to create the
-low-level HTTP client that makes requests, like the following ``GET`` request::
+low-level HTTP client that makes requests, like the following ``GET`` request:
 
-    use Symfony\Component\HttpClient\HttpClient;
+.. configuration-block::
 
-    $client = HttpClient::create();
-    $response = $client->request('GET', 'https://api.github.com/repos/symfony/symfony-docs');
+    .. code-block:: php-symfony
 
-    $statusCode = $response->getStatusCode();
-    // $statusCode = 200
-    $contentType = $response->getHeaders()['content-type'][0];
-    // $contentType = 'application/json'
-    $content = $response->getContent();
-    // $content = '{"id":521583, "name":"symfony-docs", ...}'
-    $content = $response->toArray();
-    // $content = ['id' => 521583, 'name' => 'symfony-docs', ...]
+        use Symfony\Component\HttpClient\HttpClientInterface;
 
-Performance
```

Comment:
This diff is a bit messed-up: I moved the Performance (+ cURL support) a bit down. No contents where actually removed